packages/amplication-server/src/core/prismaSchemaUtils/schema-utils.ts

```diff
@@ -206,3 +209,35 @@ export function jsonField(field: Field) {
     return EnumDataType.Json;
   }
 }
+
+export function findFkFieldNameOnAnnotatedField(field: Field): string {
+  const relationAttribute = field.attributes?.find(
+    (attr) => attr.name === "relation"
+  );
+
+  if (!relationAttribute) {
+    throw new Error(`Missing relation attribute on field ${field.name}`);
+  }
+
+  const fieldsArgs = relationAttribute.args.find(
+    (arg) => (arg.value as KeyValue).key === "fields"
+  );
+
+  if (!fieldsArgs) {
+    throw new Error(
+      `Missing fields attribute on relation attribute on field ${field.name}`
+    );
+  }
+
+  const fieldsArgsValues = (
+    (fieldsArgs.value as KeyValue).value as RelationArray
+  ).args;
+
+  if (fieldsArgsValues.length > 1) {
+    throw new Error(
+      `Relation attribute on field ${field.name} has more than one field`
```

Comment:
`Relation attribute on field ${field.name} has more than one field, which is not supported